Modules/atexitmodule.c

```diff
@@ -133,8 +151,12 @@ atexit_callfuncs(struct atexit_state *state)
         }
 
         // bpo-46025: Increment the refcount of cb->func as the call itself may unregister it
```

Comment:
The NewRef() is only needed for PyErr_FormatUnraisable() formatting, no? I don't think that we need to hold a strong reference to args+kwargs here. (Just a comment, the code is correct.)